style.css

```diff
@@ -383,6 +389,57 @@ label {
   outline: 1px solid white;
 }
 
+.popup_container {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: #c1c7d088;
+  z-index: 9999;
+  padding: 77px 5%;
+  overflow-y: scroll;
+}
+
+.popup_title {
+  margin-bottom: 20px;
+}
+
+.popup_details {
+  display: flex;
+  flex-direction: column;
+  margin-top: 12px;
+}
+
+#x-popup {
+  float: right;
+}
+
+.popup_details hr {
+  margin: 20px 0;
+}
+
+.popup_details .prog_lang {
+  margin: 0;
+}
+
+.popup_btn {
+  display: flex;
+  justify-content: center;
+  gap: 16px;
+}
+
+.popup_btn button {
+  padding: 8px 16px;
+  display: flex;
+  gap: 8px;
+  align-items: center;
+}
```

Comment:
Great job with styling 👍  But let's align the buttons to the left like on the Figma design guideline.
<img width="804" alt="Screenshot 2023-04-11 at 10 44 20 PM" src="https://user-images.githubusercontent.com/25887253/231273598-e34a0841-ea0d-4c5d-95f2-bfe58a55119c.png">
<img width="1440" alt="Screenshot 2023-04-11 at 10 52 46 PM" src="https://user-images.githubusercontent.com/25887253/231273783-9ec1f43e-86a9-4e95-a626-a402fbfd53ff.png">